core/trino-main/src/main/java/io/trino/sql/planner/DomainTranslator.java

```diff
@@ -119,7 +119,7 @@ public Expression toPredicate(TupleDomain<Symbol> tupleDomain)
         return IrUtils.combineConjuncts(toPredicateConjuncts(tupleDomain));
     }
 
-    public List<Expression> toPredicateConjuncts(TupleDomain<Symbol> tupleDomain)
+    private List<Expression> toPredicateConjuncts(TupleDomain<Symbol> tupleDomain)
```

Comment:
why?

Reply:
I had made it public for usage in DynamicPageFilter, now I don't need it so made it private again